integration-tests/src/cluster/mod.rs

```diff
@@ -166,7 +167,8 @@ impl Cluster {
                 node.account
             }
             None => {
-                let account = self.worker().dev_create_account().await?;
+                let account = utils::dev_gen_indexed(self.worker(), self.account_idx).await?;
+                self.account_idx += 1;
```

Comment:
Can we be sure this index is not used? We are using `dev_gen_indexed()` and always start from in `create_accounts()`.

Reply:
what do you mean not used? This is always used in resharing tests

Reply:
the account_idx here starts from node.len() after initial create_accounts, so no collision will ever happen

Reply:
Oh, I have not seen that line, should be fine then